### DISCUSSION TITLE
[Win11] taskbar along the left edge of the screen?

### DISCUSSION THREAD

**Discussion by bblackmoor** [Open-Shell/Open-Shell-Menu#1632] [General]
I prefer the taskbar along the left edge of the screen. I have not yet upgraded to Windows 11, because the taskbar is locked to the long edge of the screen, needlessly wasting a huge amount of space (in my opinion).

Is it likely that Open Shell will eventually restore this lost functionality, or has Microsoft made it unfeasible?

(Thank you for all of your work. <3 Open Shell.)

**Comment by bblackmoor** [comment]
I guess no one knows. But I can hope. :)

**Reply by zaadstra** [reply]
[ExplorerPatcher](https://github.com/valinet/ExplorerPatcher/releases) can do that for you.

**Comment by trexpark** [comment]
;This Key Aligns The last bar to Left 1=Left and 0=Centre
[HKEY_Current_User\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarAl"=dword:00000001

;This Key hides the Widget icon form the taskbar 1=Show 0=Hide
[HKEY_Current_User\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarDa"=dword:00000000

;This Key hides the Chat icon from the taskbar 1=ShowChat icon 0=Hide Chat icon
[HKEY_Current_User\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarMn"=dword:00000000

**Reply by bblackmoor** [reply]
Thank you, but none of that pertains to my question.

**Reply by trexpark** [reply]
Sorry, should have added, That this can be used in the meantime until there is something available. I've been deploying the above registry edits from the endpoint manager.

**Reply by bblackmoor** [reply]
I am interested in placing the taskbar (the whole taskbar, not just an icon) at the left edge of the screen. So that rather than the taskbar being along the bottom of the screen, like so:

(rest of screen is above the taskbar)
= = = = = = = = = =

It is along the left side, like so:

|| 
|| 
|| (rest of screen is to the right of the taskbar)
|| 
||
